preprocessing/lib/csvToJson.js

```diff
@@ -0,0 +1,27 @@
+const csvLoader = require('../lib/load-data');
+const cli = require('../lib/cli-common-copy');
```

Comment:
we need to fix this import please; also since the file is now in `lib/`, the requires should be `./` and not `../lib/`

Reply:
This was amended in commit 09b8135.